.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  tox:
+    uses: ./.github/workflows/tox.yml
+    secrets: inherit
+  publish:
+    needs: [Tox]
+    uses: ./.github/workflows/pypi.yml
+    secrets: inherit
+  docs:
+    needs: [PyPi]
```

Comment:
Update `pypi` to `publish` since `needs: [<job name>]`

Reply:
done